pkg/controller/installation/products/launcher/asserts_test.go

```diff
@@ -0,0 +1,83 @@
+	routev1 "github.com/openshift/api/route/v1"
+	coreosv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	k8serr "k8s.io/apimachinery/pkg/api/errors"
+	pkgclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type AssertFunc func(LauncherTestScenario, *config.Manager) error
+
+func assertNoop(LauncherTestScenario, *config.Manager) error {
+	return nil
+}
+
+func assertInstallationSuccessfullyReconciled(scenario LauncherTestScenario, configManager *config.Manager) error {
+	ctx := context.TODO()
+
+	launcherConfig, err := configManager.ReadLauncher()
+	if err != nil {
+		return err
+	}
+
+	// A namespace should have been created.
+	ns := &corev1.Namespace{}
+	err = scenario.FakeSigsClient.Get(ctx, pkgclient.ObjectKey{Name: launcherConfig.GetNamespace()}, ns)
+	if k8serr.IsNotFound(err) {
+		return errors.New(fmt.Sprintf("%s namespace should have been created", launcherConfig.GetNamespace()))
+	}
+
+	// A subscription to the product operator should have been created.
+	sub := &coreosv1alpha1.Subscription{}
+	err = scenario.FakeSigsClient.Get(ctx, pkgclient.ObjectKey{Name: defaultSubscriptionName, Namespace: launcherConfig.GetNamespace()}, sub)
+	if k8serr.IsNotFound(err) {
+		return errors.New(fmt.Sprintf("%s operator subscription was not created", defaultLauncherName))
+	}
+
+	launcherInst := &launcherv1alpha2.Launcher{}
+	err = scenario.FakeSigsClient.Get(ctx, pkgclient.ObjectKey{Name: defaultLauncherName, Namespace: launcherConfig.GetNamespace()}, launcherInst)
+	if k8serr.IsNotFound(err) {
+		return errors.New(fmt.Sprintf("%s custom resource was not created", defaultLauncherName))
+	}
+	if launcherInst.Spec.OpenShift.ConsoleURL != scenario.Installation.Spec.MasterURL {
+		return errors.New(fmt.Sprintf("%s ConsoleURL not equal MasterURL ", defaultLauncherName))
+	}
+
+	// RHSSO integration should be configured.
+	rhssoConfig, err := configManager.ReadRHSSO()
+	if err != nil {
+		return errors.New("Error getting RHSSO config")
+	}
+	kcr := &aerogearv1.KeycloakRealm{}
+	err = scenario.FakeSigsClient.Get(ctx, pkgclient.ObjectKey{Name: rhssoConfig.GetRealm(), Namespace: rhssoConfig.GetNamespace()}, kcr)
+	if !aerogearv1.ContainsClient(kcr.Spec.Clients, clientId) {
+		return errors.New(fmt.Sprintf("Keycloak client '%s' was not created", clientId))
+	}
+
+	// An Oauth client should have been created with the correct redirects.
+	launcherOauth := &oauthv1.OAuthClient{}
+	err = scenario.FakeSigsClient.Get(ctx, pkgclient.ObjectKey{Name: oauthId}, launcherOauth)
+	if k8serr.IsNotFound(err) {
+		return errors.New(fmt.Sprintf("launcher should have an Ouath Client '%s' created", oauthId))
+	}
+	launcherRoute := &routev1.Route{}
+	err = scenario.FakeSigsClient.Get(ctx, pkgclient.ObjectKey{Name: launcherRouteName, Namespace: launcherConfig.GetNamespace()}, launcherRoute)
+	if err != nil {
+		return err
+	}
+	launcherUrl := "http://" + launcherRoute.Spec.Host
```

Comment:
it's https now - fabric8-launcher/launcher-application#913

Reply:
changes aren't on the latest release v0.0.4